common/src/main/java/com/github/khanshoaib3/minecraft_access/utils/condition/Keystroke.java

```diff
@@ -9,6 +14,9 @@
  * For keys that only execute once until released and pressed again.
  */
 public class Keystroke {
+
+    protected static List<Keystroke> EXISTING_KEYSTROKES = new ArrayList<>();
```

Comment:
This is not a constant and thus should be in camelCase, not SCREAMING_SNAKE_CASE. `keystrokes` may also be a better name because "existing" doesn't really add any meaning. This field can also be final.
```suggestion
    protected static final List<Keystroke> keystrokes = new ArrayList<>();
```